### PR TITLE
converter converted result page layout

### PR DESCRIPTION
**Pull request by yoouung**
- add converted result page layout
- TODO: fix highlighting in details table
- TODO: fix highlighting bug that pdd module always activated and highlighted

### CODE REVIEW

**Comment by yoouung** [comment]
### converted result page layout
image

File: dt-demo-fe/src/pages/converter/converted.js [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+                const matches = original_highlighted_sentences.matchAll(
+                    /<mark id='([^']+)'>([^<]+)<\/mark>/g
+                );
+                if (!matches) continue;
+                for (const match of matches) {
+                    const id = match[1];
+
+                    let ostyle = "";
+                    let cstyle = "";
+                    // const idPrefixes = ["typo", "slang", "pdd", "dup", "spc"];
```
Comment (doorooful) [inline dt-demo-fe/src/pages/converter/converted.js:127]: Please remove unnecessary line

**Review by doorooful — APPROVED** [review]
Thanks for your effort !!

File: dt-demo-fe/src/pages/converter/converted.js (hunk 1 above)
Comment (yoouung) [reply dt-demo-fe/src/pages/converter/converted.js]: oops...!! thanks for check 🫶🏻

**Pull request merged**